### PR TITLE
Allow using CMD as shortcut modifier

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by IntelOrca** [comment]
Code looks okay but I am unable to confirm if it works.

**Comment by marijnvdwerf** [comment]
Setting/using CMD works. Unfortunately, the same keycode is used for the windows key, so this is going to need some platform-specific adjustments. Also, as noted by @pizza2004 "ALT +" should actually be "OPTION +".

Some issues:
- If you set a shortcut to `CTRL + CMD + F`, both the game shortcut as well as the fullscreen shortcut are used.
- Default keyboard combinations still use CTRL
- Y-axis dragging still uses CTRL.
- Different issue: keyboard shortcuts probably need to be rethought to be used on OS X. As most devices only have a small keyboard

**Comment by marijnvdwerf** [comment]
@IntelOrca Can you confirm my expectation that this messes up on Windows?

**Comment by IntelOrca** [comment]
You should wrap it in an ```#if defined(__APPLE__) && defined(__MACH__)```

**Comment by IntelOrca** [comment]
You can do this for ALT as well and write a different string.

**Comment by marijnvdwerf** [comment]
Done. Also added conditionals to make the default shortcuts use CMD instead of CTRL.

File: data/language/english_uk.txt [hunk 1]
```diff
@@ -3971,6 +3971,8 @@ STR_5629    :Difficulty level
 STR_5630    :Enable unlocking of scenarios
 STR_5631    :Original DLC Parks
 STR_5632    :Build your own...
+STR_5633    :CMD + 
+STR_5634    :OPTION +
```
Comment (janisozaur) [inline data/language/english_uk.txt]: @Gymnasiast pointed out when you add new strings, you should file a new issue with localisation repo.

**Comment by janisozaur** [comment]
@marijnvdwerf you can add this change to [changelog](https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/changelog.txt)

**Comment by marijnvdwerf** [comment]
They do ;)

File: data/language/english_uk.txt (hunk 1 above)
Comment (marijnvdwerf) [inline data/language/english_uk.txt]: Done. OpenRCT2/Localisation#360 and OpenRCT2/Localisation#359

**Comment by marijnvdwerf** [comment]
@Broxzier Better?

**Comment by Broxzier** [comment]
@marijnvdwerf: I do like this better, yes.

**Comment by AaronVanGeffen** [comment]
Seems to be working nicely on OS X for me, too. :+1:

**Comment by marijnvdwerf** [comment]
Still feels weird to see 'OPTION + CMD + L' instead of '⌥⌘L'. Guess that's another issue though.

**Pull request merged**